### PR TITLE
update notes content for v4

### PR DESCRIPTION
**Pull request by johngruen**
This pull request updates significant chunks of the experiment content for Notes. I've also flattened out the folder structure for the experiment images b/c the old structure was actually left over from our django days (believe it or not).

### CODE REVIEW

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3482/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/39f1ec973148953fbc09ac916bdff6d504f4745d/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3482/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/6412326a7d1038cdcd959d0e33a207be3981ec85/

**Comment by meandavejustice** [comment]
Restarted integration tests, they were failing because of a fluke.

Also pulled down, looks good to me. Thanks for cleaning up that image directory, @johngruen.

This is a :+1: from me. but we should wait for @flodolo to take a look.

File: locales/en-US/experiments.ftl [hunk 1]
```diff
@@ -72,15 +72,17 @@ nomore404sContributors1Title = Developer, Wayback Machine, The Internet Archive
 nomore404sContributors2Title = Director, Wayback Machine, The Internet Archive
 nomore404sContributors3Title = Senior Crawl Engineer, The Internet Archive
 notesDescription = A simple notepad built into Firefox. Ever need to jot something down while you browse the web? This is the experiment for you.
-notesIntroduction = Notes adds a simple, one-page notepad to your browser’s sidebar. We’ve added a few basic formatting options to start, but we want your feedback to tell us what features you’d like to see us add.
+notesIntroductionMultinote = Notes adds a simple notepad to your browser’s sidebar. Sign into your Firefox Account to sync all of your notes with end-to-end encryption.
```
Comment (flodolo) [inline locales/en-US/experiments.ftl]: Sign into ----> Sign in to
https://design.firefox.com/photon/copy/word-list.html#s

**Review by flodolo — CHANGES_REQUESTED** [review]
One fix to a string (and fix the typo in the commit message since you're amending it)

File: locales/en-US/experiments.ftl (hunk 1 above)
Comment (johngruen) [reply locales/en-US/experiments.ftl]: Good catch, thanks!

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3482/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/858dec73b3d97b1954daad941ea45360c165f5e8/

**Review by chuckharmston — APPROVED** [review]
Still blocked on an r+ from Francesco, but this looks good to me.

**Review by flodolo — APPROVED** [review]

**Pull request merged**